src/WebApp/flask/templates/analytics.html

```diff
@@ -87,6 +94,13 @@ <h3>Cluster Information</h3>
         <div>           
             <button type="submit" id="create-project-button" class="btn btn-primary" onclick="return confirm('Are you sure you want to delete the cluster?')">Delete Cluster</button>
         </div>
+        <div>
+            {% if clusterDetails.Status == "DeletionFailed" %}           
+            <small class="text-danger">
+                Cluster Creation Failed : {{clusterDetails.Message}}
```

Comment:
do not capitalize every word, unless it's a name of a brand, azure service, or some technology.

Cluster creation failed.